packages/web/src/pages/audio-transactions-page/AudioTransactionsPageProvider.tsx

```diff
@@ -0,0 +1,354 @@
+    metadata: {}
+  },
+  {
+    signature: 'banana_bread',
+    transactionType: TransactionType.TRENDING_REWARD,
+    method: TransactionMethod.RECEIVE,
+    date: '6/23/2020',
+    change: '5',
+    balance: '1000000',
+    metadata: undefined
+  },
+  {
+    signature: 'banana_bread',
+    transactionType: TransactionType.TIP,
+    method: TransactionMethod.RECEIVE,
+    date: '6/23/2020',
+    change: '100',
+    balance: '100000',
+    metadata: {}
+  },
+  {
+    signature: 'banana_bread',
+    transactionType: TransactionType.PURCHASE,
+    method: TransactionMethod.STRIPE,
+    date: '6/23/2020',
+    change: '100',
+    balance: '100200',
+    metadata: {
+      discriminator: TransactionMetadataType.PURCHASE_SOL_AUDIO_SWAP,
+      usd: '0',
+      sol: '0',
+      audio: '0',
+      purchaseTransactionId: '0',
+      swapTransactionId: '0'
+    }
+  },
+  {
+    signature: 'banana_bread',
+    transactionType: TransactionType.TIP,
+    method: TransactionMethod.SEND,
+    date: '6/23/2020',
+    change: '-100',
+    balance: '100000',
+    metadata: {}
+  },
+  {
+    signature: 'banana_bread',
+    transactionType: TransactionType.TRENDING_REWARD,
+    method: TransactionMethod.RECEIVE,
+    date: '6/23/2020',
+    change: '5',
+    balance: '1000000',
+    metadata: undefined
+  },
+  {
+    signature: 'banana_bread',
+    transactionType: TransactionType.CHALLENGE_REWARD,
+    method: TransactionMethod.RECEIVE,
+    date: '6/23/2020',
+    change: '1',
+    balance: '333333333',
+    metadata: undefined
+  },
+  {
+    signature: 'banana_bread',
+    transactionType: TransactionType.TIP,
+    method: TransactionMethod.RECEIVE,
+    date: '6/23/2020',
+    change: '100',
+    balance: '100000',
+    metadata: {}
+  },
+  {
+    signature: 'banana_bread',
+    transactionType: TransactionType.TIP,
+    method: TransactionMethod.SEND,
+    date: '6/23/2020',
+    change: '-100',
+    balance: '100000',
+    metadata: {}
+  },
+  {
+    signature: 'banana_bread',
+    transactionType: TransactionType.TRENDING_REWARD,
+    method: TransactionMethod.RECEIVE,
+    date: '6/23/2020',
+    change: '5',
+    balance: '1000000',
+    metadata: undefined
+  },
+  {
+    signature: 'banana_bread',
+    transactionType: TransactionType.TIP,
+    method: TransactionMethod.RECEIVE,
+    date: '6/23/2020',
+    change: '100',
+    balance: '100000',
+    metadata: {}
+  }
+]
+
+const AudioTransactionsPage = g((props) => {
+  const { userId, goToRoute } = props
+
+  const onClickRow = useCallback((trackRecord: any) => {}, [])
+
+  const onSort = (sorters: any) => {}
+
+  const isEmpty = entries.length === 0
+
+  const childProps = {
+    description: messages.description,
+    dataSource: entries,
+    entries,
+    goToRoute,
+    isEmpty,
+    loading: false,
+    onClickRow,
+    onSort,
+    title: messages.title,
+    userId
+  }
+
+  return <props.children key={userId} {...childProps} />
+})
+
+const makeMapStateToProps = () => {
```

Comment:
I would be in favor of avoiding the Provider pattern and instead using redux hooks directly in `AudioTransactionsPage`

Also I'm not sure this is currently working because `makeMapStateToProps` is never called